hack/create-infra-kubeconfig.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+SERVICE_ACCOUNT_NAME=$1
+
+TOKEN_NAME=$(kubectl get serviceaccount kubevirt-csi -o json | jq '.secrets[0].name' | xargs echo)
```

Comment:
That will not work with k8s 1.24, due to https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/2799-reduction-of-secret-based-service-account-token#kep-2799-reduction-of-secret-based-service-account-tokens

Reply:
Thanks for the comment, I have converted this to issue, so we can handle it,
This was already merged as part of another change